### PR TITLE
Vlan networks fix, Isol Cpu Fix for numa and DPDK

### PR DESCRIPTION
**Pull request by ahmadfsbd**
- Updated the deprecated parameters for numa

- Fixed CPU cores calculations for NUMA and OVS-DPDK

- Log details for cpu and nics on numa nodes

### CODE REVIEW

File: src/pilot/dell_nfv.py [hunk 1]
```diff
@@ -252,8 +258,14 @@ def edit_environment_files(
                     '\\" |" ' +
                     dpdk_file)
                 cmds.append(
-                    'sed -i "s|# IsolCpusList:.*|IsolCpusList: ' +
-                    self.nfv_params.isol_cpus + '|" ' + dpdk_file)
+                    'sed -i "s|IsolCpusList:.*|IsolCpusList: \\"' +
+                    self.nfv_params.isol_cpus + '\\" |" ' + dpdk_file)
+
+            # Change neuton-network-type to vlan. In future if we
```
Comment (heydog15) [inline src/pilot/dell_nfv.py:264]: neuton-network-type should be neutron-network-type

File: src/pilot/deploy-overcloud.py [hunk 2]
```diff
@@ -603,7 +603,7 @@ def main():
         env_opts += " -e ~/pilot/templates/overcloud/environments/" \
                     "network-isolation.yaml" \
                     " -e ~/pilot/templates/network-environment.yaml" \
-                    " -e /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovs.yaml" \
+                    " -e /home/stack/pilot/templates/overcloud/environments/services/neutron-ovs.yaml" \
```
Comment (heydog15) [inline src/pilot/deploy-overcloud.py:606]: Why don't we do ~/pilot instead of /home/stack/pilot?

File: src/pilot/dell_nfv.py (hunk 1 above)
Comment (ahmadfsbd) [reply src/pilot/dell_nfv.py]: Changed

File: src/pilot/deploy-overcloud.py (hunk 2 above)
Comment (ahmadfsbd) [reply src/pilot/deploy-overcloud.py]: Good catch. Fixed it

File: src/pilot/dell_nfv.py [hunk 3]
```diff
@@ -218,13 +218,14 @@ def edit_environment_files(
                 if ovs_dpdk:
                     dpdk_nics = self.find_ifaces_by_keyword(nic_env_file,
                                                             'Dpdk')
+                    logger.info("DPDK-NICs >>" + str(dpdk_nics))
```
Comment (gaelrehault) [inline src/pilot/dell_nfv.py:221]: Could you change to logger.debug or remove ?

File: src/pilot/nfv_parameters.py [hunk 4]
```diff
@@ -188,6 +189,7 @@ def get_isol_cpus(self):
         isol_cpus = sorted(map(int, isol_cpus), key=int)
         isol_cpus = self.range_extract(isol_cpus)
         self.isol_cpus = ','.join(map(str, isol_cpus))
+        logger.info("isol CPUs >> " + str(self.isol_cpus))
```
Comment (gaelrehault) [inline src/pilot/nfv_parameters.py:192]: debug

File: src/pilot/dell_nfv.py (hunk 3 above)
Comment (ahmadfsbd) [reply src/pilot/dell_nfv.py]: Resolved.

**Comment by ahmadfsbd** [comment]
This PR is good to go. Please review and merge.

**Pull request merged**